src/PrestaShopBundle/Service/Cache/Refresh.php

```diff
@@ -50,46 +49,56 @@ public function __construct($env = 'prod')
     {
         umask(0000);
         set_time_limit(0);
-        $this->env = $env;
-        $this->output = new NullOutput();
+        $this->env = _PS_MODE_DEV_ ? 'dev' : 'prod';
         $this->commands = [];
 
         require_once _PS_ROOT_DIR_.'/app/AppKernel.php';
-        $kernel = new \AppKernel($this->env, false);
+        $kernel = new \AppKernel($this->env, _PS_MODE_DEV_);
```

Comment:
Yep, leave it to false please. The stack traces are not sent to the user and the dev shops can't be installed on many mutual servers.